pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -235,33 +243,81 @@ func nodesWherePreemptionMightHelp(nodes []*framework.NodeInfo, m framework.Node
 	return potentialNodes
 }
 
+// calculateNumCandidates returns the number of candidates the findCandidates
+// method must produce from dry running based on the constraints given by
+// maxCandidatesAsPercentageOfNodes and minCandidates.
+func calculateNumCandidates(numNodes int) int {
+	if numNodes < minCandidates || maxCandidatesAsPercentageOfNodes >= 100 {
```

Comment:
This conditional is not adding anything that the lines below can't handle. We can just save the branching.

Reply:
Once we standardize the args, I'd expect the validation logic (maxCandidatesAsPercentageOfNodes >= 100) to be put elsewhere.

BTW: `if numNodes < minCandidates` is already handled in `calculateOffset()`, right?